lib/config_parser.rb

```diff
@@ -2,23 +2,32 @@
 
 require 'fluent/config/v1_parser'
 require 'optparse'
+require_relative 'config_pb'
 
 # Accepts a file path and prints out parsed version
 class ConfigParser
   def initialize(argv = ARGV)
     @argv = argv
+    @ruby_parse = nil # default - won't parse ruby code in a config file
     prepare_input_parser
     input_validation
     @file_parse = parse_configuration
-    print_configuration(@file_parse)
+    @proto_obj = proto_configuration(@file_parse)
+    File.write('../json/config.json', Config::Directive.encode_json(@proto_obj))
```

Comment:
Just to give @davidbtucker on some context here if you are going to review it - I had an offline discussion with Mihika on where to output data so another process can read it. As this is an file used internally so I think where to put it is up to us. 

I think a special directory `json` is fine for now when Mihika and I discussed it - though I know normally I would expect this file will be written in a runtime environment.